books/free-programming-books-id.md

```diff
@@ -21,6 +21,7 @@
   * [Angular](#angular)
   * [Deno](#deno)
 * [Vue](#vue)
+* [Flutter](#flutter)
```

Comment:
Fix list alphabetize-lists lint rule moving between 

https://github.com/EbookFoundation/free-programming-books/blob/699d51c27291eff63d2a86a02e5fb45494d4bce9/books/free-programming-books-id.md#L7-L8

Do the same with related section 😉